src/components/views/rooms/EventTile.js

```diff
@@ -290,6 +290,18 @@ module.exports = WithMatrixClient(React.createClass({
 
         var left = 0;
 
+        var readReceiptData = Object.create(null);
+        var room = this.props.matrixClient.getRoom(this.props.mxEvent.getRoomId());
+        if (room) {
+            // [ {type/userId/data} ]
+            room.getReceiptsForEvent(this.props.mxEvent).forEach(function(r) {
```

Comment:
I'm confused. I have not changed the interface to this component.

I dislike how `this.props.readReceipts` aren't read receipts at all, and are in fact a list of `RoomMembers`, which is why I have to contort like this. Are you asking for me to modify the interface to this component to make `this.props.readReceipts` actually be read receipts with an additional `roomMember` field?